regolith/schemas.json

```diff
@@ -2876,6 +2876,34 @@
             "anyof_type": ["string", "integer"]
         }
     },
+    "packages": {
+        "_id": {
+            "description": "short representation, such as this-is-package-name",
+            "required": true,
+            "type": "string"
+        },
+        "active": {
+            "description": "status of package, such as active or outdated.",
```

Comment:
This is a bool, so it can't be active or outdated.....  replace with "the status whether the package is active or not"